src/components/auth/signup-form.tsx

```diff
@@ -145,6 +145,46 @@ export function SignupForm() {
               </FormItem>
             )}
           />
+          <FormField
+            control={form.control}
+            name="confirmPassword"
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel className="text-foreground text-sm">
+                  Confirm Password
+                </FormLabel>
+                <div className="relative">
+                  <Lock className="text-muted-foreground pointer-events-none absolute top-1/2 left-3 h-4 w-4 -translate-y-1/2" />
+                  <FormControl>
+                    <Input
+                      {...field}
+                      type={showPassword ? 'text' : 'password'}
+                      autoComplete="new-password"
+                      placeholder="min 8 characters"
+                      className="bg-muted/20 focus-visible:ring-primary/30 placeholder:text-muted-foreground/60 h-11 rounded-xl pr-12 pl-10 transition focus-visible:ring-2"
+                    />
+                  </FormControl>
+                  <Button
```

Comment:
If you are controlling showing and hiding password using the "pasword" field, you dont need another button in confirmPassword to hide and show password. You can control both fields using the button in "password".